Denik CZ.js

```diff
@@ -0,0 +1,110 @@
+function detectWeb(doc, url) {
```

Comment:
Save in Scaffold, then look in your translator directory for the file. https://www.zotero.org/support/dev/translators#translator_structure